postgresql/_database.sls

```diff
@@ -26,7 +26,7 @@ postgresql_database_{{ svr_name|default('localhost') }}_{{ database_name }}:
     - name: {{ database.get('name', database_name) }}
     - encoding: {{ database.encoding }}
     - user: postgres
-    - template: template0
+    - template: template1
```

Comment:
FYI @atengler as this will change default behavior (to use what's postgres default which is template1) instead of hard-coded template0.